### PR TITLE
[JENKINS-70693] Call `FilePathDynamicContext.createContextualObject(FilePath, FlowNode)` 

### PR DESCRIPTION
**Pull request by timja**
Related to issue [[JENKINS-70693]](https://issues.jenkins-ci.org/browse/JENKINS-70693)

Verified in the acceptance test harness that this fixes the test failing there.
https://github.com/jenkinsci/acceptance-test-harness/blob/master/src/test/java/plugins/ExternalWorkspaceManagerPluginTest.java

Alternative to https://github.com/jenkinsci/external-workspace-manager-plugin/pull/76

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1 +1,5 @@
-buildPlugin()
+buildPlugin(
+  configurations: [
+    [platform: 'linux', jdk: 17],
```
Comment (timja) [inline Jenkinsfile:3]:
```suggestion
```

**Review by jglick — APPROVED** [review]
Thanks!

**Comment by bolt-alex-somai** [comment]
Thank you, @timja, for handling this! I've tested it locally, and it works. Previously I was able to reproduce the issue.

May I merge it?

Also, now that incrementals are enabled for the plugin, do I have to do anything special to release it after the merge? A while ago, I used to run `mvn release:prepare release:perform` locally.

**Review by bolt-alex-somai — APPROVED** [review]

**Comment by alexsomai** [comment]
Thank you, @timja, for handling this! I've tested it locally, and it works. Previously I was able to reproduce the issue.

May I merge it?

Also, now that incrementals are enabled for the plugin, do I have to do anything special to release it after the merge? A while ago, I used to run `mvn release:prepare release:perform` locally.

**Review by alexsomai — APPROVED** [review]

**Comment by timja** [comment]
> Also, now that incrementals are enabled for the plugin, do I have to do anything special to release it after the merge? A while ago, I used to run mvn release:prepare release:perform locally.

Incrementals changes nothing for release, if you want it to be automated you would have to follow: https://www.jenkins.io/doc/developer/publishing/releasing-cd/

> May I merge it?

Yes go for it

**Pull request merged**